### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by JULIETYCL**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Consequently, the base URL would change to
 
 4. Once the server is up and running, you can start using the app. For ease of testing, follow these steps: 
 
-(1) post call: `python3 app.py post` (this call will return a drs object id, e.g. 12345)
+(1) post call: `action post` (this call will return a drs object id, e.g. 12345)
```
Comment (AlexanderSenf) [inline README.md]: Can you run this command as 
`action post` instead of `python3 app.py post`?

File: action.py [hunk 2]
```diff
@@ -1,10 +1,9 @@
-import json
-
 import click
+import json
 import requests
 
-base_url = "http://172.27.46.118:8080/ga4gh/drs/v1/objects"
-base_info_url = "http://172.27.46.118:8080/ga4gh/drs/v1/service-info"
+base_url = "http://172.20.144.25:8080/ga4gh/drs/v1/objects"
```
Comment (AlexanderSenf) [inline action.py:5]: Are these IPs fixed, or will they change every time you restart your computer?

You could also make this part of the CLI - provide a command line option for the server base IP, and provide a default setting, such as 'localhost' or '127.0.0.1', etc. That way a user can run it regardless of where the server is by providing the URL as a parameter.

File: README.md (hunk 1 above)
Comment (JULIETYCL) [reply README.md]: Yes. That is why I updated the readme.

File: action.py (hunk 2 above)
Comment (JULIETYCL) [reply action.py:5]: I will do!

**Comment by JULIETYCL** [comment]
Approved

**Review by JULIETYCL — COMMENTED** [review]
Changed the commands for calling the services.

**Pull request closed without merging**